<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgEditExe</class>
 <widget class="QDialog" name="dlgEditExe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SiPAR - Edição de Exercícios</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lblKey">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>36</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Sigla:</string>
   </property>
   <property name="buddy">
    <cstring>qleKey</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="qleKey">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="lblDescr">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>12</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição:</string>
   </property>
   <property name="buddy">
    <cstring>qleDescr</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="qleDescr">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>10</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="lblPAR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sítio PAR:</string>
   </property>
   <property name="buddy">
    <cstring>cbxPAR</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblAnv">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>50</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Aeronave:</string>
   </property>
   <property name="buddy">
    <cstring>cbxAnv</cstring>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbxVento">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>270</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Vento</string>
   </property>
   <widget class="QLabel" name="lblVentoVel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Intensidade:</string>
    </property>
    <property name="buddy">
     <cstring>qsbVentoVel</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="qsbVentoVel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>84</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="specialValueText">
     <string>Unknown</string>
    </property>
    <property name="suffix">
     <string> kt</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblVentoDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Direção:</string>
    </property>
    <property name="buddy">
     <cstring>qsbVentoDir</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="qsbVentoDir">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>102</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> gr</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>61</number>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbxGate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>270</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Gate</string>
   </property>
   <widget class="QLabel" name="lblGateDist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Distância:</string>
    </property>
    <property name="buddy">
     <cstring>qsbGateDist</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="qsbGateDist">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="specialValueText">
     <string>Unknown</string>
    </property>
    <property name="suffix">
     <string> NM</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblGateAfst">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>140</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Afastamento do eixo:</string>
    </property>
    <property name="buddy">
     <cstring>qsbGateAfst</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="qsbGateAfst">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> m</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblGateAlt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>43</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Altura:</string>
    </property>
    <property name="buddy">
     <cstring>qsbGateAlt</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="qsbGateAlt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> ft</string>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>3000</number>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cbxPAR">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>45</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxAnv">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>45</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="lblCab">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cabeceira:</string>
   </property>
   <property name="buddy">
    <cstring>cbxCab</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxCab">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>45</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qleKey</tabstop>
  <tabstop>qleDescr</tabstop>
  <tabstop>cbxPAR</tabstop>
  <tabstop>cbxCab</tabstop>
  <tabstop>cbxAnv</tabstop>
  <tabstop>qsbGateDist</tabstop>
  <tabstop>qsbGateAlt</tabstop>
  <tabstop>qsbGateAfst</tabstop>
  <tabstop>qsbVentoVel</tabstop>
  <tabstop>qsbVentoDir</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgEditExe</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgEditExe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
